app/templates/community/index.hbs

```diff
@@ -18,6 +18,10 @@
       <a href="http://stackoverflow.com/questions/tagged/ember.js">StackOverflow</a> &mdash; used to track questions. Just tag your question with <code>ember.js</code> or <a href="http://stackoverflow.com/questions/tagged/ember.js">search for questions with that tag</a>. Please check to see if your question has already been answered before asking a new one.
     </p>
 
+    <p>
+      <a href="https://dev.to/t/ember" target="_blank">DEV's Ember Community</a> &mdash; a great place to find interesting articles about best practices, application architecture and new learnings on <code>ember.js</code>
```

Comment:
```suggestion
      <a href="https://dev.to/t/ember" target="_blank" rel="noopener noreferrer">DEV's Ember Community</a> &mdash; a great place to find interesting articles about best practices, application architecture and new learnings. Post your articles with the tag <code>ember</code>.
```

Was thinking this could be an improvement, what do you think?

Reply:
P.S. Here's the rule https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/link-rel-noopener.md related to the Travis CI failure https://travis-ci.org/ember-learn/ember-website/builds/597289177#L508-L509

Reply:
This absolutely makes sense. Thanks, I'll make the correction and push the changes accordingly.